target/linux/ath79/image/generic.mk

```diff
@@ -421,6 +421,16 @@ define Device/comfast_cf-e314n-v2
 endef
 TARGET_DEVICES += comfast_cf-e314n-v2
 
+define Device/comfast_cf-e375ac
+  SOC := qca9563
+  DEVICE_VENDOR := COMFAST
+  DEVICE_MODEL := CF-E375AC
+  DEVICE_PACKAGES := kmod-usbcore kmod-usb2 kmod-ath10k-ct \
```

Comment:
BTW: you don't mention USB in your specifications.

Reply:
There's no USB port available in this particular model.
I kept the kmod-usb2 because the ar71xx target has this definition.

Reply:
I've removed the kmod-usb2 too.
I think COMFAST has an outdoor version of this device that has an USB port.

Reply:
Yes, if there is no usb port, we don't need to install this _by default_.